### ISSUE TITLE
Angular & quaggaJS problems

### ISSUE THREAD

**Issue by grahlie** [serratus/quaggaJS#60]
Hello!

I having problem using quagga in a angular application, the quagga reader does work when visiting the blank page without angular.

But used in a angular application I don't get any reply from the quagga "onDetected".

Do anyone have any suggestions on this problem?

**Comment by serratus** [comment]
Hi there,
how do you start quaggaJS in your application? Do you have some sample code? Do you see any errors/warnings in the console? Is everything hooked up properly, and do you see the video?

**Comment by grahlie** [comment]
Im using the file_input example with angular and angulars routeProvider my example code is pretty simple just adding the different javascript needed for angular.

put up a simple fiddle here: https://jsfiddle.net/t3tou02m/

and in order.html i have the same html code as in the file_input example as said before. I dont get anything in the console I'm afraid so that dont help me so much. And visiting the page using angular www.example.com/#/order the quagga script dont fire but when visiting the page without angular www.example.com/pages/order.html the quagga script works.

**Comment by serratus** [comment]
Sorry for the long delay. I've been really busy lately.
Alright, so I looked at your example and I'm trying to make sense of it. So you say that quagga isn't even loaded when visiting the route? It should be loaded on the first request only, since it's present in the `<head/>` part. Did you include my version of `file_input.js`? What markup does the  `order.html` template contain? If you still need some help on that, I'll send you a fiddle in the next couple of days.

**Comment by grahlie** [comment]
I understand you have been working on other things as well no worries.

But here is a more detailed example at least I think:
http://codepen.io/anon/pen/xwVzwx

In "order.html" page displays the file_input.html form. And then the file_input.js file is original from example. All files included from Quagga script is original from that example.

**Comment by JohnPaulHarold** [comment]
I've got Quagga working in an Angular app (still massively experimental and ugly).  I've got a button that `init`s Quagga and I see my webcam surface with the green rects dancing around.  I didn't do anything clever, just treated it as a Global object.  That might not be an Angular-ish way of doing things.  I too get no `onDetected()` calls, but then I think I'm suffering from only having access to poor/low quality cameras and Quagga can't get a solid detection.
